docssrc/src/permissions.md

```diff
@@ -117,3 +117,82 @@ Now we declare our command with arguments. We use a `PlayerArgument` and apply t
 > As you can see, there are multiple ways of applying permissions to commands with arguments. In the `/god` command shown above, the permission was applied to the whole command. In the `/kill` command shown above, the permission was applied to the argument.
 >
 > There's not really much difference between the two methods, but I personally would use _argument permissions_ as it has greater control over arguments.
+
+-----
+
+## Child Base Permissions
+### Example - /economy command with argument permissions
+Child Based Permissions allow you to group permissions together, we achieve this by laying out our permission groups in the `plugin.yml` file which Bukkit registers as valid permissions. When the CommandAPI checks if our player has a permission Bukkit considers if they have the child of a permission as well. This not only keeps permissions easier to manage, it also makes your code cleaner and gives you a nice place to layout out all of your permissions, what they do and what other permissions inherit them.
```

Comment:
"Child-based permissions" (I want a dash between "Child" and "based", and I want "based" and "permissions" to all be in lowercase)

Reply:
> Child Based Permissions allow you to group permissions together, <-- 

Let's make this a single sentence. Replace this comma with a full stop, and make the following "We" start with an uppercase letter.

Reply:
> When the CommandAPI checks if our player has a permission Bukkit considers

Add a comma between "permission" and "Bukkit" in this sentence